litex/soc/software/bios/complete.c

```diff
@@ -0,0 +1,174 @@
+// This file is Copyright (c) 2020 Franck Jullien <franck.jullien@gmail.com>
+//
+//     Largely inspired/copied from U-boot and Barebox projects wich are:
+//         Sascha Hauer, Pengutronix, <s.hauer@pengutronix.de>
+
+// License: BSD
```

Comment:
Since this is mostly there, could you use an SPDX identifier tag here?

Reply:
ok